pkg/controller/context.go

```diff
@@ -83,6 +84,9 @@ type Context struct {
 	FieldManager string
 	// RESTConfig is the loaded Kubernetes apiserver rest client configuration
 	RESTConfig *rest.Config
+	// Scheme is the Kubernetes scheme that should be used when serialising and
+	// deserialising API objects
+	Scheme *runtime.Scheme
```

Comment:
I can't see where this new field is used.

Reply:
I think this because the original PR was rebased many times and this was not required in the end?
I'll remove this.